service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/objects/EncounterHelper.java

```diff
@@ -0,0 +1,18 @@
+package uk.nhs.adaptors.oneoneone.cda.report.objects;
```

Comment:
Let's not use such "helpers". I think it would be a better idea to extract necessary mappers calls out of EncounterMapper and only pass their outcome if necessary.